### PR TITLE
pamfile: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/pamfile.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# pamfile
+
+> Describe Netpbm (.pam or .pnm) files.
```
Comment (kbdharun) [inline pages/common/pamfile.md:3]:
```suggestion
> Describe Netpbm (PAM or PNM) files.
```
I would suggest capitalizing extensions in the page description.

File: pages/common/pamfile.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/pamfile.html>.
+
+- Describe the specified Netpbm files:
+
+`pamfile {{path/to/file1}} {{path/to/file2}}`
```
Comment (kbdharun) [inline pages/common/pamfile.md:8]:
```suggestion
`pamfile {{path/to/file1 path/to/file2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by reinhart1010 — APPROVED** [review]
LGTM, thanks for contribution!

**Pull request merged**